source/about-mongosync.txt

```diff
@@ -157,6 +157,30 @@ If the reverse sync is successful, ``mongosync`` enters the ``RUNNING`` state.
 The synchronization continues in the reverse direction from the original 
 sync job. 
 
+.. _mongosync-considerations:
+
+Considerations for Continuous Sync
+----------------------------------
+
+Avoid using ``mongosync`` for disaster recovery for the following reasons:
+
+- For the destination cluster to have an up to date, consistent view of the 
+  source cluster’s data, ``mongosync`` must :ref:`commit <c2c-api-commit>` . 
+  If a disaster occurs and shuts down the source cluster before ``mongosync`` 
```

Comment:
We can update to:
"If the source cluster shuts down before mongosync can commit (e.g., due to a disaster scenario), the cluster won't [...]"